script/munin-httpd

```diff
@@ -59,8 +59,6 @@ sub handle_request
 	}
 }
 
-package main;
-
 $ENV{PATH} = '/usr/bin:/bin';
```

Comment:
I prefer to revert to `package main;` to obviously indicate that the above code was in a package.
It could be moved to another file, or the end of this one, but it all feels not as sweet as currently.

Reply:
cgzones: what do you think?